tests/BladeXTest.php

```diff
@@ -17,9 +17,9 @@ public function it_can_register_a_single_component_with_providing_a_view_and_com
 
         $registeredComponents = BladeX::getRegisteredComponents();
 
-        $this->assertCount(1, $registeredComponents);
-        $this->assertEquals('myView1', $registeredComponents[0]->name);
-        $this->assertEquals('registerDirectoryTest/myView1', $registeredComponents[0]->bladeViewName);
+        $this->assertCount(2, $registeredComponents);
```

Comment:
Needed to increment all these assertions by one. This was quite tedious, what if we want to add another first-party component in the future?

Reply:
Maybe we can introduce some the concept of component types and not let `BladeX::getRegisteredComponents` return internal ones?